classes/components/forms/statistics/users/ReportForm.inc.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * @file classes/components/form/statistics/users/ReportForm.inc.php
+ *
+ * Copyright (c) 2014-2020 Simon Fraser University
+ * Copyright (c) 2000-2020 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class AssignToIssueForm
+ * @ingroup classes_controllers_form
+ *
+ * @brief A preset form for configuring the users report.
+ */
+namespace PKP\components\forms\statistics\users;
+use \PKP\components\forms\{FormComponent, FieldOptions};
+
+class ReportForm extends FormComponent {
+	/**
+	 * Constructor
+	 *
+	 * @param string $action URL to submit the form to
+	 * @param \Context $context The context
+	 */
+	public function __construct(string $action, \Context $context)
```

Comment:
We're currently using inline `{` syntax for function declarations.

Reply:
Ok! I've just spotted the current style when there's return types:
```
function test() : void {
}
```